projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -1031,6 +1040,64 @@ public function register_post_meta() {
 		register_meta( 'post', '_jetpack_post_was_ever_published', $jetpack_post_was_ever_published );
 	}
 
+	/**
+	 * Create a Subscribers menu displayed on self-hosted sites.
+	 *
+	 * - It is not displayed on WordPress.com sites.
+	 * - If you have stats and the new Stats experience enabled, it directs you there.
+	 * - If you do not, it directs you to Calypso to the existing Subscribers page.
+	 *
+	 * @return void
+	 */
+	public function add_subscribers_menu() {
+		/*
+		 * Do not display any menu on WoA and WordPress.com Simple sites.
+		 * They already get a menu item under Users via nav-unification.
+		 */
+		if ( ( new Host() )->is_wpcom_platform() ) {
+			return;
+		}
+
+		$status = new Status();
+
+		/*
+		 * Do not display if we're in Offline mode,
+		 * or if the user is not connected.
+		 */
+		if (
+			$status->is_offline_mode()
+			|| ! ( new Connection_Manager( 'jetpack' ) )->is_user_connected()
+		) {
+			return;
+		}
+
+		$domain = $status->get_site_suffix();
+
+		if (
+			( new Modules() )->is_active( 'stats' )
+			&& Stats_Options::get_option( 'enable_odyssey_stats' )
+		) {
+			$stats_dashboard = new Stats_Dashboard();
+			$hook            = add_submenu_page(
+				'jetpack',
+				__( 'Subscribers', 'jetpack' ),
+				__( 'Subscribers', 'jetpack' ),
+				'view_stats',
+				'stats#!/stats/subscribers/' . $domain,
+				array( $stats_dashboard, 'render' )
+			);
+			add_action( "load-$hook", array( $stats_dashboard, 'admin_init' ) );
+		} else {
+			add_submenu_page(
+				'jetpack',
+				esc_attr__( 'Subscribers', 'jetpack' ),
```

Comment:
is `esc_attr` necessary?  Should we do it in [the other call](https://github.com/Automattic/jetpack/pull/31718/files#diff-776cd74bed19fb81f284b9dbfd8258e57d5fb0a5db07c0e1db94e47dbb0d6182R1083) too?

Reply:
Good call. Done in ce5ad18308fa65561f29f684b2ffe3461e9c45ae